ext/mbstring/mbstring.c

```diff
@@ -60,7 +60,7 @@
 # include "php_onig_compat.h"
 # include <oniguruma.h>
 # undef UChar
-#elif HAVE_PCRE || HAVE_BUNDLED_PCRE
+#elif HAVE_BUNDLED_PCRE
```

Comment:
This should be removed entirely instead. `if HAVE_PCRE || HAVE_BUNDLED_PCRE` => `if 1` => always true.